Makefile

```diff
@@ -85,7 +34,7 @@ DRIVERS_SRCS = netif-driver.c
 DRIVERS_SRCS_UTIL = bdring.c pktmem.c physmmap.c res-create.c
```

Comment:
Not directly related to this patch, but it should be `:=`  instead of `=` here (this applies in most other places also). See https://www.gnu.org/software/make/manual/make.html#Reading-Makefiles for details.

Reply:
Fixed in the update.